src/templates/base/2019/base_chapter.html

```diff
@@ -61,13 +61,23 @@
 {# Calls to action for readers who want to engage more with this chapter. #}
 {% macro render_actions() %}
   <a class="alt btn" href="https://discuss.httparchive.org/t/{{ metadata.get('discuss') }}">
-    <img src="/static/images/discuss.png" alt="{{ self.discuss_this_chapter() }}" height="24" width="24" loading="lazy" /><span id="num_comments"></span> <span data-translation id="comment-singular">{{ self.comment() }}</span><span data-translation id="comment-plural">{{ self.comments() }}</span>
+    <svg width="18" height="18">
```

Comment:
We've lost the alt tag here. Can we add it back with a title tag as otherwise VoiceOver reads the 4 digit link:
```suggestion
    <svg width="18" height="18">
    <title>{{ self.discuss_this_chapter() }}</title>
```